src/python/e2e-test-runner/e2e_test_runner/test_kafka.py

```diff
@@ -0,0 +1,108 @@
+import copy
+import os
+import uuid
+from typing import Any
+
+import pytest
+from confluent_kafka import Consumer, Producer  # type: ignore
+
+IS_LOCAL = not os.environ["KAFKA_BOOTSTRAP_SERVERS"].startswith("SASL_SSL")
```

Comment:
I know i told you to do this, but I'm not sure I love this. Hmm. Well, it's fine for now. I guess all this comment is saying is "huh maybe we should revisit in the future or something"